### ISSUE TITLE
ember 1.12 support

### ISSUE THREAD

**Issue by givanse** [pzuraq/liquid-tether#13]
I don't think `1.12` should be supported, in the long term, but I needed it for a project. So, I made a branch for that, in your `package.json` use:

    "liquid-tether": "givanse/liquid-tether#1.12"

Maybe you would like to merge that branch, not into master of course. Just to keep the branch within the same project for the benefit of any others that still have `1.12` apps.

**Comment by pzuraq** [comment]
Agreed that Ember 1.12 support is not a long term goal for this library. I think it would be beneficial to keep a branch for 1.12 around for a while though, as long as you or others would be willing to maintain it. I'll take a look at the branch when I get some time, see what I can do!

**Comment by givanse** [comment]
Alright, I can keep an eye on that branch for a while. Just create the branch in the repo so I can submit the PR for it.

**Comment by pzuraq** [comment]
I created the branch `ember-1.12` to avoid confusion over which version the branch is referring, PR away

**Comment by pzuraq** [comment]
@givanse still waiting on the PR, ember 1.12 support is not a priority for me so I'm going to close this issue. Feel free to reopen if you'd like to add it.
